test_util/test_upgrade_vpc.py

```diff
@@ -115,6 +115,9 @@ def main():
     stable_installer_url = os.environ['STABLE_INSTALLER_URL']
     installer_url = os.environ['INSTALLER_URL']
 
+    config_yaml_override_install = os.getenv('CONFIG_YAML_OVERRIDE_INSTALL')
+    config_yaml_override_upgrade = os.getenv('CONFIG_YAML_OVERRIDE_UPGRADE')
```

Comment:
according to [`DcosApiInstall::genconf`](https://github.com/BenWhitehead/dcos/blob/9afb4fc00bc5190d4257c768901cba34f94f280f/test_util/installer_api_test.py#L143) the `add_config_path` arg represents a yaml that will override any previously specified values. I think it's important to keep the override in the name so that people know it's the highest priority of any passed in value(s).